src/components/WebPlayBack.tsx

```diff
@@ -1,3 +1,4 @@
+import { Console } from 'console';
```

Comment:
Are we using this import? We should be able to just call `console.log` from anywhere

Reply:
No the Team was not. Thank you.